cypress/e2e/about.cy.ts

```diff
@@ -0,0 +1,79 @@
+import enUS from '~/i18n/locales/en.json'
```

Comment:
I see the checks are failing. So I checked if it was the bug. But also failed on my device. I think because we do not have the `cypress.config.ts` setup to import files this way you should change it here to `import enUS from "../../i18n/locales/en.json"`. When I did this locally all the checks passed for the tests your wrote.